### ISSUE TITLE
Lutris fails to change runners for a game.

### ISSUE THREAD

**Issue by Nilot352** [lutris/lutris#1411]
this is the output for `lutris -d`

> Traceback(most recent call last):
  File "/usr/lib/python3/dist-packages/lutris/gui/config_dialogs.py", line 376, in on_save
    self.game.save()
  File "/usr/lib/python3/dist-packages/lutris/game.py", line 173, in save
    playtime=self.playtime,
  File "/usr/lib/python3/dist-packages/lutris/pga.py", line 262, in add_or_update
    return add_game(**params)
  File "/usr/lib/python3/dist-packages/lutris/pga.py", line 225, in add_game
    return sql.db_insert(PGA_DB, "games", game_data)
  File "/usr/lib/python3/dist-packages/lutris/util/sql.py", line 54, in db_insert
    field_values
  File "/usr/lib/python3/dist-packages/lutris/util/sql.py", line 30, in cursor_execute
    return cursor.execute(query, params)
sqlite3.IntegrityError: UNIQUE constraint failed: games.id

**Comment by 7oxicshadow** [comment]
Glad someone else has posted this. I discovered this problem recently but could not be sure that it wasn't a problem that I had created because I am manually modifying the database via a script.

When you look at the code, it appears to check for an existing entry and records the index. But instead of calling db_update it calls db_insert. When you try to insert a record using an existing entry number you get the "sqlite3.IntegrityError: UNIQUE constraint failed: games.id" error.
